compose/config/config_schema_v2.0.json

```diff
@@ -270,6 +271,7 @@
             "name": {"type": "string"}
           }
         },
+        "labels": {"$ref": "#/definitions/list_or_dict"},
```

Comment:
Now that config_schema_v2.1.json has been merged, we can move this property there.